GMG/Core/Models/ScoreFactory/ScoreFactory.swift

```diff
@@ -94,11 +97,22 @@ final class ScoreFactory {
             let newChordCell: ChordCell = ChordCell(
                 chord: firstChordCell.chord,
                 chordCandidates: firstChordCell.chordCandidates,
-                startTime: .zero
+                startTime: .zero,
+                duration: .zero
             )
             filteredChordCells = [newChordCell] + filteredChordCells[1...]
         }
 
+        /// 최소 코드 길이
+        let minimumChordDuration: TimeInterval = 5 * 0.03
```

Comment:
제나가 공유해준 기준을 보니 ChordCell의 텍스트는 전체 너비의 약 8% 미만일 때 잘리는 형태로 디자인되어 있더라고요.

그렇다면 최소 코드 길이도 동일한 비율을 반영해 5 * 0.08 정도로 맞추는 것이 더 일관성 있을 것 같습니다!

View 레벨에서는 ChordCell이 다음/이전 세그먼트와 이어지는 경우 너비 비율이 8% 미만이면 텍스트가 보이지 않도록 처리해두겠습니다

```suggestion
        let minimumChordDuration: TimeInterval = 5 * 0.08
```

Reply:
반영하겠습니다~

Reply:
네 감사합니다~